roles/speedtest-tracker/tasks/main.yml

```diff
@@ -8,7 +8,7 @@
 
 - name: Speedtest-Tracker Docker Container
   docker_container:
-    name: speedtest_tracker
+    name: speedtest-tracker
```

Comment:
If someone is already running it, this change will cause the new container to fail to start. 

A step is required prior to this to stop/remove the old one - which is arguably more mess than just leaving this how it is.